cfg/rationalize.go

```diff
@@ -38,6 +38,11 @@ func Rationalize(c *Config) error {
 		return err
 	}
 
+	c.GcsAuth.TokenUrl, err = decodeURL(c.GcsAuth.TokenUrl)
+	if err != nil {
+		return err
+	}
```

Comment:
nit: How about using the short-if idiom:
```suggestion
	if c.GcsAuth.TokenUrl, err = decodeURL(c.GcsAuth.TokenUrl); err != nil {
		return err
	}
```